cmd/validate/image.go

```diff
@@ -37,7 +37,7 @@ import (
 	"github.com/enterprise-contract/ec-cli/internal/format"
 	"github.com/enterprise-contract/ec-cli/internal/output"
 	"github.com/enterprise-contract/ec-cli/internal/policy"
-	"github.com/enterprise-contract/ec-cli/internal/policy/source"
+	source_utils "github.com/enterprise-contract/ec-cli/internal/policy/source"
```

Comment:
I would not make this change, if the shadowing of `source` is an issue at all, I'd rename the variable from `source` to `src` or just `s`. e.g. [here](https://github.com/yashvardhannanavati/ec-cli/blob/dadfa454e854a6fcd71d02a560a69f5b130d7cf9/cmd/validate/image.go#L230)

Reply:
makes sense. Done